core/modules/qdisp/testQDisp.cc

```diff
@@ -193,8 +198,8 @@ BOOST_AUTO_TEST_SUITE(Suite)
     // resource object for all chuncks has been properly constructed. We use
     // the same chunkID for all tests (see setRName() below).
     //
-    SequentialInt sequence(0);
-    int chunkId = 1234, millisInt = 200;
+    int chunkId = 1234;
+    int millisInt = 20000;
```

Comment:
I was getting a timeout during a unit test. I also changed the timeout function so it would return in 0.1 seconds of success.

Reply:
Understood. In those cases when the test values are not just the random ones (like the chunkId number) it would probably make a sense to document a reason in the testing code. Some simple comment: "...intervals which are less than (say) 0.1 second may cause the timeouts and may break the unit test...".